<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRichTextDialog</class>
 <widget class="QDialog" name="EditRichTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Положение</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="EditLocationWidget" name="widgetLocation" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Текст</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="writingSystem">
            <enum>QFontDatabase::Any</enum>
           </property>
           <property name="currentFont">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPointSize">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QToolButton" name="toolButtonBold">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/Resources.qrc">
               <normaloff>:/TextEdit/22/format-text-bold.png</normaloff>:/TextEdit/22/format-text-bold.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonItalic">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/Resources.qrc">
               <normaloff>:/TextEdit/22/format-text-italic.png</normaloff>:/TextEdit/22/format-text-italic.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonUnderline">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/Resources.qrc">
               <normaloff>:/TextEdit/22/format-text-underline.png</normaloff>:/TextEdit/22/format-text-underline.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonStrikethrough">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/Resources.qrc">
               <normaloff>:/TextEdit/22/format-text-strikethrough.png</normaloff>:/TextEdit/22/format-text-strikethrough.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QToolButton" name="toolButtonTextColor">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/Resources.qrc">
               <normaloff>:/TextEdit/22/TextColor.png</normaloff>:/TextEdit/22/TextColor.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonBackroundColor">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/Resources.qrc">
               <normaloff>:/TextEdit/22/BrushColor.png</normaloff>:/TextEdit/22/BrushColor.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditRichText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Гипербаза</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="EditHyperbaseWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Инфо</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditLocationWidget</class>
   <extends>QWidget</extends>
   <header>EditLocationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditHyperbaseWidget</class>
   <extends>QWidget</extends>
   <header>EditHyperbaseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditRichTextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditRichTextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
